diagnostic.md

```diff
@@ -7,40 +7,48 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+Check to make sure the repository is where i want it and not inside of another repository.
+Then create my own branch.
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git branch response
+git checkout response
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git status
 ```
 
 5.We want to prepare a change for a commit by adding a file to the staging
     area. What command do we use to stage a file named `diagnostic.md`?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git add diagnostic.md
 ```
 
 6.Once `diagnostic.md` is staged, we have to make a commit by `git commit`.
 What are the two formatting items you **need** to make up your commit message?
 
-<!-- Remove this comment and place your answer here. -->
-
+I need to specify the file or directory i want to commit. The commit message
+needs to have a subject line, first word capitalized, that succinctly describes
+the changes I made. Next is a blank line. On the next line I can go into greater
+detail about the commit if I need to.
 -7.Should you ever edit published history?
 
- <!-- Remove this comment and place your answer here. -->
+ I dont think that would be a good practice, and it seems like it should be
```

Comment:
Note: Commits will not be discarded if the branch is deleted, which is a fairly common thing to do